### PR TITLE
add monitoring logging as addons

### PR DESCRIPTION
**Pull request by w13915984028**
```
add addons to harvester-configuration

update logging monitoring as addons
    
      update logging as addons
      update monitoring as addons
      update addons list in v1.2.0
```

Signed-off-by: Jian Wang <w13915984028@gmail.com>

issue: 
https://github.com/harvester/harvester/issues/3423

backend code PR: https://github.com/harvester/harvester-installer/pull/402

Epic: https://github.com/harvester/harvester/issues/3262

### CODE REVIEW

File: docs/advanced/addons.md [hunk 1]
```diff
@@ -16,4 +16,12 @@ v1.1.0 currently ships with two Addons:
 * [pcidevices-controller](./pcidevices.md)
 * [vm-import-controller](./vmimport.md)
 
-![](/img/v1.2/addons/DefaultAddons.png)
\ No newline at end of file
+![](/img/v1.2/addons/DefaultAddons.png)
+
+_Available as of v1.2.0_
+
+v1.2.0 ships two new Addons:
```
Comment (vickyhella) [inline docs/advanced/addons.md:23]:
```suggestion
v1.2.0 ships with two more Addons:
```

File: docs/install/harvester-configuration.md [hunk 2]
```diff
@@ -517,6 +522,36 @@ install:
...
+```
+
+There are four addons as of Harvester v1.2.0 release:
+
+```
+  harvester-vm-import-controller
+  harvester-pcidevices-controller
+  rancher-monitoring
+  rancher-logging
+```
```
Comment (vickyhella) [inline docs/install/harvester-configuration.md:553]:
```suggestion

- harvester-vm-import-controller
- harvester-pcidevices-controller
- rancher-monitoring
- rancher-logging

```

File: docs/install/harvester-configuration.md [hunk 3]
```diff
@@ -517,6 +522,36 @@ install:
...
+```yaml
+install:
+  addons:
+    rancher-monitoring:
+      enabled: true
+    rancher-logging:
+      enabled: false
+```
+
+There are four addons as of Harvester v1.2.0 release:
```
Comment (vickyhella) [inline docs/install/harvester-configuration.md:546]:
```suggestion
Harvester v1.2.0 ships with four addons:
```

File: docs/install/harvester-configuration.md (hunk 2 above)
Comment (vickyhella) [reply docs/install/harvester-configuration.md:553]: Use a list rather than wrapping them in the code block

File: docs/advanced/addons.md (hunk 1 above)
Comment (vickyhella) [reply docs/advanced/addons.md:23]: `v1.2.0 ships two new Addons:` seems like v1.2.0 only has two addons, but it actually has four

File: docs/logging/logging.md [hunk 4]
```diff
@@ -21,6 +21,16 @@ It is important to know what is happening/has happened in the `Harvester Cluster
 The size of logging data is related to the cluster scale, workload and other factors. `Harvester` does not use persistent storage to store log data inside the cluster. Users need to set up a log server to receive logs accordingly.
 :::
 
+_Available as of v1.2.0_
+
+The logging feature becomes addons and is disabled by default in new installation.
```
Comment (vickyhella) [inline docs/logging/logging.md:26]:
```suggestion
The logging feature is now implemented with an addon and is disabled by default in new installations.
```

Comment (vickyhella) [reply docs/logging/logging.md:26]: Maybe `Addon` here should be in singular form? Correct me if I'm wrong though~

File: docs/logging/logging.md [hunk 5]
```diff
@@ -21,6 +21,16 @@ It is important to know what is happening/has happened in the `Harvester Cluster
...
+
+User can enable/disable logging addons [addons](../advanced/addons.md) from Harvester WebUI after installation.
```
Comment (vickyhella) [inline docs/logging/logging.md:28]:
```suggestion
Users can enable/disable the `rancher-logging` [addon](../advanced/addons.md) from the Harvester WebUI after installation.
```

Comment (vickyhella) [reply docs/logging/logging.md:28]: - duplicate `addon`
- better give the full name of the addon

File: docs/logging/logging.md [hunk 6]
```diff
@@ -21,6 +21,16 @@ It is important to know what is happening/has happened in the `Harvester Cluster
...
+
+User can enable/disable logging addons in Harvester installation via customized [harveter-configuration](../install/harvester-configuration.md#installaddons) file.
```
Comment (vickyhella) [inline docs/logging/logging.md:30]:
```suggestion
Users can also enable/disable the `rancher-logging` addon in their Harvester installation by customizing the [harveter-configuration](../install/harvester-configuration.md#installaddons) file.
```

File: docs/logging/logging.md [hunk 7]
```diff
@@ -21,6 +21,16 @@ It is important to know what is happening/has happened in the `Harvester Cluster
...
+
+For the Harvester clusters upgraded from v1.1.*, the logging is converted to addons automatically and kept enabled as before.
```
Comment (vickyhella) [inline docs/logging/logging.md:32]:
```suggestion
For Harvester clusters upgraded from v1.1.*, the logging feature is converted to an addon automatically and kept enabled as before.
```

File: docs/monitoring/monitoring.md [hunk 8]
```diff
@@ -6,6 +6,18 @@ title: "Monitoring"
 
 _Available as of v0.3.0_
 
+The monitoring feature is enabled by default.
+
+_Available as of v1.2.0_
+
+The monitoring feature becomes addons and is disabled by default in new installation.
```
Comment (vickyhella) [inline docs/monitoring/monitoring.md:13]:
```suggestion
The monitoring feature is now implemented with an addon and is disabled by default in new installations.
```

File: docs/monitoring/monitoring.md [hunk 9]
```diff
@@ -6,6 +6,18 @@ title: "Monitoring"
...
+
+User can enable/disable monitoring addons [addons](../advanced/addons.md) from Harvester WebUI after installation.
```
Comment (vickyhella) [inline docs/monitoring/monitoring.md:15]:
```suggestion
Users can enable/disable the `rancher-monitoring` [addon](../advanced/addons.md) from the Harvester WebUI after installation.
```

File: docs/monitoring/monitoring.md [hunk 10]
```diff
@@ -6,6 +6,18 @@ title: "Monitoring"
...
+
+User can enable/disable monitoring addons in Harvester installation via customized [harveter-configuration](../install/harvester-configuration.md#installaddons) file.
```
Comment (vickyhella) [inline docs/monitoring/monitoring.md:17]:
```suggestion
Users can also enable/disable the `rancher-monitoring` addon in their Harvester installation by customizing the [harveter-configuration](../install/harvester-configuration.md#installaddons) file.
```

File: docs/monitoring/monitoring.md [hunk 11]
```diff
@@ -6,6 +6,18 @@ title: "Monitoring"
...
+
+For the Harvester clusters upgraded from v1.1.*, the monitoring is converted to addons automatically and kept enabled as before.
```
Comment (vickyhella) [inline docs/monitoring/monitoring.md:19]:
```suggestion
For Harvester clusters upgraded from v1.1.*, the monitoring feature is converted to an addon automatically and kept enabled as before.
```

**Review by vickyhella — COMMENTED** [review]
Just some nits

**Review by vickyhella — APPROVED** [review]

**Review by ibrokethecloud — APPROVED** [review]
lgtm. thanks.

**Review by bk201 — APPROVED** [review]
LGTM, thanks!

**Review by guangbochen — APPROVED** [review]

**Pull request merged**